SHMEM/isx.c

```diff
@@ -890,26 +890,27 @@ static void create_permutation_array()
     permute_array[i] = i;
   }
 
-  shuffle(permute_array, NUM_PES, sizeof(int));
+  shuffle(permute_array, NUM_PES);
 }
 
 /*
  * Randomly shuffles a generic array
```

Comment:
Might be a good idea to update the comment to read "Randomly shuffles an array of PE ids"